roaring/roaring.go

```diff
@@ -818,34 +822,51 @@ type errWriter struct {
 	w   io.Writer
 	err error
 	n   int
+
+	byte2 []byte
```

Comment:
Should you just make these arrays to not have pointers? They're fixed size anyways

Reply:
Ah if they're arrays you can't use the binary functions...